packages/utils/src/carequality/patient-discovery-statistics.ts

```diff
@@ -0,0 +1,24 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { getXcpdStatisticsForPatient } from "@metriport/core/external/carequality/pd/get-statistics";
+import { getEnvVarOrFail } from "../../../api/src/shared/config";
+
+const apiUrl = getEnvVarOrFail("API_URL");
+const cxId = getEnvVarOrFail("CX_ID");
+const sqlDBCreds = getEnvVarOrFail("DB_CREDS");
```

Comment:
Can we add a comment here to remind the developer to use the read replica, please?